### PR TITLE
feat(feedback): implement feedback section.

### PR DESCRIPTION
**Pull request by LeVHoangduc**

### CODE REVIEW

File: practice-one/src/assets/css/feedback.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.feedbacks {
```
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/feedback.css:1]: What is this? block, section, or something else?

File: practice-one/src/assets/css/feedback.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  margin-top: 208px;
+}
+.feedbacks__container {
+  width: 100%;
+  max-width: 1120px;
```
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/feedback.css:6]: Why we have this max-width?

File: practice-one/src/assets/css/feedback.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  display: flex;
+  gap: 40px;
+}
+.feedback {
+  box-sizing: border-box;
+  width: 540px;
+  height: 280px;
+  flex-shrink: 0;
+  background: #fff;
```
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/feedback.css:15]: Code style

File: practice-one/src/assets/css/feedback.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+.feedback__header {
+  display: flex;
+  justify-content: space-between;
+}
+.feedback__photo {
+  width: 100px;
+  height: 100px;
+  flex-shrink: 0;
+}
+.feedback__header--intro {
```
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/feedback.css:28]: intro is not status or property, right?

File: practice-one/src/assets/css/feedback.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:1]: Hi Hung, this is block.

File: practice-one/src/assets/css/feedback.css (hunk 3 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:15]: i will fix it.

File: practice-one/src/assets/css/feedback.css (hunk 2 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:6]: I thought so to ensure the length of the element. But, this is my mistake, that's redundant! I will fix it.

File: practice-one/src/assets/css/feedback.css (hunk 4 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:28]: i think i will change it to "feedback__introduction" .

File: practice-one/src/assets/css/feedback.css (hunk 3 above)
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/feedback.css:15]: Code style

Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:15]: i will update it

File: practice-one/src/assets/css/feedback.css (hunk 1 above)
Comment (hungnguyenhuu) [reply practice-one/src/assets/css/feedback.css:1]: So what is this block using for?

File: practice-one/src/assets/css/feedback.css (hunk 4 above)
Comment (hungnguyenhuu) [reply practice-one/src/assets/css/feedback.css]: That's good

File: practice-one/src/assets/css/feedback.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:1]: I use this block for including all the elements contained in the feedback section.

Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:1]: @hungnguyenhuu

Comment (hungnguyenhuu) [reply practice-one/src/assets/css/feedback.css:1]: So why you need to split into 2 pieces feedbacks and feedbacks__container?

Comment (LeVHoangduc) [reply practice-one/src/assets/css/feedback.css:1]: I got your idea. it means redundant classname. I will fix it.

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu i have updated it. Please help me to check it again

**Pull request merged**